lib/perl/Genome/Model/Tools/DetectVariants2/Manta.pm

```diff
@@ -0,0 +1,120 @@
+package Genome::Model::Tools::DetectVariants2::Manta;
+
+use warnings;
+use strict;
+
+use Genome;
+
+use Genome::Sys::LSF::ResourceParser qw(parse_lsf_params);
+
+use File::Spec qw(join);
+use Data::Dumper;
+
+class Genome::Model::Tools::DetectVariants2::Manta {
+    is => 'Genome::Model::Tools::DetectVariants2::Detector',
+    has_param => [
+        lsf_resource => {
+            default_value => Genome::Config::get('lsf_resource_dv2_manta'),
+        },
+    ],
+};
+
+sub _gb_ram {
+    my $self = shift;
+
+    my $lsf_params = parse_lsf_params($self->lsf_resource);
+
+    my $kb_ram = $lsf_params->{'rLimits'}->{'RSS'};
+
+    if (!defined($kb_ram)) {
+        $self->fatal_message('Unable to resolve RAM from the lsf resource string: '. $self->lsf_resource);
+    }
+
+    return ($kb_ram / 1_000_000);
+}
+
+sub _detect_variants {
+    my $self = shift;
+
+    my $working_directory = $self->_temp_staging_directory;
+
+    # Optional params defined as detector params in the processing profile
+    # parse the param string to make the necessary param to hash ref translation
+    # Example : --exome --config-file=/my/foo/config.txt
+    # ( exome => 1, config_file => '/my/foo/config/txt' )
+    # Also see Genome::Model::Tools::DetectVariants2::Mutect::parse_params
+    my @resolved_config_params = (
+        '--tumor-bam-file' => $self->aligned_reads_input,
+        '--version' => $self->version,
+        '--working-directory' => $working_directory,
+        '--reference-fasta' => $self->reference_sequence_input,
+    );
+
+    # Determine which manta output VCF should be used as the "high-quality" result
+    # If single-sample, assume tumor-only (NOT GERMLINE).  We know somatic if a normal is defined.
+    my $manta_vcf_filename = 'tumorSV.vcf.gz';
+
+    if ( defined($self->control_aligned_reads_input) ) {
+        push @resolved_config_params, ('--normal-bam-file' => $self->control_aligned_reads_input);
+        $manta_vcf_filename = 'somaticSV.vcf.gz';
+    }
+
+    my @dv2_config_params = split(' ',$self->params);
```

Comment:
This isn't necessarily safe.  For example, if one of the values is a quoted string with a space in it.

Why not loop over the class's known properties (or ask the class what properties it has), extract their values, and build up a hash of params to pass to ```create()``` on line 65?  Something like:
```
my @dv2_config_params = map { $_ => $self->$_ } @known_params_list;
my $config_cmd = Genome::Model::Tools::Manta::Config->create(@dv2_config_params, @resolved_config_params);
```